tests/integration_tests/test_model_save_and_load.py

```diff
@@ -32,6 +32,79 @@
 )
 
 
+def test_model_load_from_checkpoint(tmpdir, csv_filename, tmp_path):
```

Comment:
❤️ @geoffreyangus Thank you -- so cool!  For my edification, which checkpoint are we comparing?  It looks like the loaded model from storage (`ludwig_model_2`) is the result of training `ludwig_model_1` all the way through the `1` epoch -- as opposed to some intermediate checkpoint (e.g., saved after a few steps).  Is this correct?  Thank you!

Reply:
yeah that is correct– it's just the latest checkpoint deposited into training_checkpoints/ during the course of training. In the 1 epoch case, it is equivalent to the model weights at the end of training.

For some reason on my local, the models weren't equivalent after 2 epochs– my hunch is that this is because there was a difference between the "best" checkpoint (loaded at the end of training) and the "latest" checkpoint

Reply:
🙇